CWE/Common/Extensions.cs

```diff
@@ -61,5 +68,97 @@ public static string GetUsernameAndTag(this SocketUser socketUser)
         {
             return socketUser.Username + "#" + socketUser.Discriminator;
         }
+
+        /// <summary>
+        /// Makes a <see cref="EmbedAuthorBuilder"/> from a <see cref="IUser"/>.
+        /// </summary>
+        /// <param name="user">The user to make an <see cref="EmbedAuthorBuilder"/> from</param>
+        /// <returns>A <see cref="EmbedAuthorBuilder"/> whos properties are set to the current users.</returns>
+        public static EmbedAuthorBuilder GetAuthorEmbed(this IUser user)
+        {
+            return new EmbedAuthorBuilder()
+            {
+                Name = $"{user.Username}{user.Discriminator}",
```

Comment:
```suggestion
                Name = $"{user.Username}#{user.Discriminator}",
```
Add the # between the username and discriminator.